src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -167,7 +166,7 @@ Qgs3DMapCanvasWidget::Qgs3DMapCanvasWidget( const QString &name, bool isDocked )
 
   mOptionsMenu->addSeparator();
 
-  mActionSync2DNavTo3D = new QAction( tr( "2D Map View Follows 3D Camera" ), this );
+  mActionSync2DNavTo3D = new QAction( tr( "2D map view follows 3D camera" ), this );
```

Comment:
Actually actions SHOULD have title case -- it's just checkboxes and label widgets which need sentence case!

Reply:
My bad. This is fixed.